.github/pull_request_template.md

```diff
@@ -2,6 +2,13 @@
 
 # Changes
 
-- 
+-
+# Checklist
+
+- [ ] Applicable documentation updated (guides, quickstart, postman collections, tutorial, fidesdemo)
+- [ ] Good unit test/integration test coverage
```

Comment:
I like the intention here. If we were to make this less subjective, what would that look like? Something like "Overall test coverage increased" maybe?

Reply:
yes we can definitely rephrase - I guess I was thinking that not every PR needs tests / is testable, and when I hear "increase overall test coverage" it makes me think of increasing percentage, but if we added new code, and added new tests we probably maintained the percentage.

Reply:
Sure thing! This will serve as a good reminder regardless, so let's merge it